pkg/utils/pod/pod_utils_test.go

```diff
@@ -104,36 +104,43 @@ func TestAddLWSVariables(t *testing.T) {
 		name                     string
 		pod                      *corev1.Pod
 		expectedLwsLeaderAddress string
+		expectedGroupSize        string
 	}{
 		{
 			name:                     "Leader pod",
-			pod:                      testutils.MakePodWithLabels("test-sample", "0", "", "default"),
+			pod:                      testutils.MakePodWithLabels("test-sample", "0", "", "default", 2),
```

Comment:
I'd have one case where you having different than two..

Reply:
Hey, Kevin. I don't quite understand what this suggestion means. Could you explain it more clearly?

Reply:
use a different value from "2".

Reply:
done